### ISSUE TITLE
How to connect to Dremio Cloud?

### ISSUE THREAD

**Issue by JorisTruong** [sodadata/soda-core#2007]
Hi,

I am trying to try out `soda-core` with Dremio Cloud. According to [Soda's documentation](https://docs.soda.io/soda/connect-dremio.html#connection-configuration-reference), the connection configuration should look like this:
```
data_source my_datasource_name:
  type: dremio
  host: 127.0.0.1
  port: 5432
  username: simple
  password: simple_pass
  schema: public
```

The [Dremio's documentation](https://docs.dremio.com/cloud/sonar/client-apps/python#config-file-options) suggests to use `data.dremio.cloud` as the host and a token instead of username/password.

Any advice? I also have installed the ODBC driver for Arrow Flight SQL as it looks like this is what is used the the [source code](https://github.com/sodadata/soda-core/blob/70b8753d24f9a8e9de5912874eaf9ed5dc60538c/soda/dremio/soda/data_sources/dremio_data_source.py#L65), but it has not been working for me so far.

Here is a code snippet with the error log:
```
scan = Scan()
scan.set_data_source_name("cost_analysis")
scan.add_configuration_yaml_str(
    """
    data_source cost_analysis:
      type: dremio
      host: data.dremio.cloud
      port: 443
      token: <personal access token>
"""
)
scan.add_sodacl_yaml_str(sodacl_yaml_str)
scan.execute()
scan.assert_no_checks_fail()
```

`Encountered a problem while trying to connect to dremio: can only concatenate str (not "int") to str`

**Comment by tools-soda** [comment]
SAS-2800

**Comment by m1n0** [comment]
Hi! This is currently not supported, but it should be a fairly straightforward change. Would you be open to help and contribute token support?

**Comment by JorisTruong** [comment]
Closed as #2009 has been merged
